draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -2267,20 +2487,20 @@ might be a 100Gb/s trunk port shared by ten 10GE client ports.
 
 The controller, when asked to set a 10GE client connectivity service
 between two muxponder's client ports, needs first to check whether
-there is already an existing WDM tunnel between the two muxponders
+there is already an existing DWDM tunnel between the two muxponders
 and then take different actions:
 
-1. if the WDM tunnel already exists, the controller needs only to
+1. if the DWDM tunnel already exists, the controller needs only to
 enable the 10GE client ports to establish the 10GE client
 connectivity service;
 
-2. if the WDM tunnel does not exist, the controller has to first
-establish the WDM tunnel, finding a proper optical path matching
+2. if the DWDM tunnel does not exist, the controller has to first
+establish the DWDM tunnel, finding a proper optical path matching
```

Comment:
I-D.ietf-ccamp-wdm-tunnel-yang defines the 'WDM tunnel' model (and not the 'DWDM tunnel' model)

```suggestion
2. if the WDM tunnel does not exist, the controller has to first
establish the WDM tunnel, finding a proper optical path matching
```